src/frontend/pages/_app.tsx

```diff
@@ -27,36 +28,41 @@ declare global {
   }
 }
 
-const resolveRoute = function resolveRoute (url: URL): string {
-  const pathname = url.pathname
+const resolveRoute = function resolveRoute(url: URL): string {
+  const pathname = url.pathname;
 
   if (pathname.startsWith('/product')) {
-    return '/product/{product-id}'
+    return '/product/{product-id}';
   }
 
   if (pathname.startsWith('/api/products/')) {
-    return '/api/products/{product-id}'
+    return '/api/products/{product-id}';
   }
 
   if (pathname.startsWith('/cart/checkout/')) {
-    return '/cart/checkout/{order-id}'
+    return '/cart/checkout/{order-id}';
   }
 
-  return pathname
-}
+  return pathname;
+};
 
 if (typeof window !== 'undefined') {
+  Bugsnag.start({
+    apiKey: window.ENV.BUGSNAG_API_KEY,
```

Comment:
Instead of using this key, we need to add a new one, `FRONTEND_BUGSNAG_API_KEY`, similar to [what Adolfo did for the backend](https://github.com/bugsnag/opentelemetry-demo/pull/6/files#diff-e9cbb0224c4a3d23a6019ba557e0cd568c1ad5e1582ff1e335fb7d99b7a1055dR107). 
The main Bugsnag API key is the organization one and for error monitoring we need to use a project one.

Reply:
Should it be changed for BugsnagPerformance also?

Reply:
Yes, let's keep it consistent.